SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -82,5 +82,9 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/** Change 'Load' value in inspector to represent worker Frame Time instead of a fraction of target FPS.*/
 	UPROPERTY(EditAnywhere, config, Category = "Metrics", meta = (ConfigRestartRequired = false))
 	bool bUseFrameTimeAsLoad;
+
+	/** Server Worker Type Names. Edit the launch configuration description to affect this property. */
+	UPROPERTY(VisibleAnywhere, config, Category = "Access Control", meta = (ConfigRestartRequired = false))
```

Comment:
They can't :D it's read only if it's `VisibleAnywhere`

Reply:
Nice, do we even want it to be visible?

Reply:
It may be ok to have it visible, in case users need this information in blueprint. I think we should hide it for now and make it visible first when we have a usecase though

Reply:
I'll see how we can hide it